tests/test_pipenv.py

```diff
@@ -227,6 +227,36 @@ def test_uninstall(self):
             c = p.pipenv('run python -m requests.help')
             assert c.return_code > 0
 
+    @pytest.mark.files
+    @pytest.mark.run
+    @pytest.mark.uninstall
+    def test_uninstall_all_local_files(self):
+        file_name = 'tablib-0.12.1.tar.gz'
+        # Not sure where travis/appveyor run tests from
+        test_dir = os.path.dirname(os.path.abspath(__file__))
+        source_path = os.path.abspath(os.path.join(test_dir, 'test_artifacts', file_name))
+
+        with PipenvInstance() as p:
+            shutil.copy(source_path, os.path.join(p.path, file_name))
+            os.mkdir(os.path.join(p.path, "tablib"))
+
+            c = p.pipenv('install {}'.format(file_name))
+            key = [k for k in p.pipfile['packages'].keys()][0]
```

Comment:
I should have caught this when the original test was added. Why are we creating a new list for keys when keys() already returns a list?

Let's not continue this trend into new tests.

Reply:
Probably because in Python 2 `keys()` returns a `list` but in Python 3 `keys()` returns a _view_ that would need to be coerced to a `list` to be indexed.

Reply:
we do this in a number of places in the codebase for this reason and in the other tests that check on this as well

Reply:
OK rather than deal with this silly 2/3 issue I just removed the copy/pasta of the offending test.  The original test remains but this new one does not propagate that behavior.